master/internal/trial.go

```diff
@@ -935,6 +1091,11 @@ func (t *trial) processContainerRunning(
 
 	t.containers[msg.Container.ID] = msg.Container
 	t.containerAddresses[msg.Container.ID] = msg.ContainerStarted.Addresses
+	// TODO(DET-5280): Uncomment this code, nuke below it.
```

Comment:
I don't think you are linking to the correct ticket number here. Besides, can we just have a global switch in the master to choose between V1 and V2. I don't want the V2 logic to be heavily dependent on V1 logic. It would be bad for writing tests and hard to switch to V2 cleanly.

Reply:
> I don't think you are linking to the correct ticket number here.

I just linked the push arch epic.

>  I don't want the V2 logic to be heavily dependent on V1 logic.

It's not dependent at all, once v2 is on, v1 will not exist at all.

Reply:
done